vllm/model_executor/layers/quantization/torchao.py

```diff
@@ -1,23 +1,34 @@
 # SPDX-License-Identifier: Apache-2.0
+import os
 from typing import Any, Optional
 
 import torch
 import torch.nn.functional as F
 from torch.nn.parameter import Parameter
 
+from vllm.logger import init_logger
 from vllm.model_executor.layers.linear import (LinearBase, LinearMethodBase,
                                                UnquantizedLinearMethod)
 from vllm.model_executor.layers.quantization import QuantizationMethods
 from vllm.model_executor.layers.quantization.base_config import (
     QuantizationConfig, QuantizeMethodBase)
 from vllm.model_executor.utils import set_weight_attrs
 
+logger = init_logger(__name__)
+
 
 class TorchAOConfig(QuantizationConfig):
     """Config class for torchao."""
 
     def __init__(self, torchao_config) -> None:
         self.torchao_config = torchao_config
+        # TODO (jerryzh168): enable after 2.8.0
+        # TorchAO quantization relies on tensor subclasses. In order,
+        # to enable proper caching this needs standalone compile
+        # os.environ["VLLM_TEST_STANDALONE_COMPILE"] = "1"
+        # logger.info("Using TorchAO: Setting VLLM_TEST_STANDALONE_COMPILE=1")
+        os.environ["VLLM_DISABLE_COMPILE_CACHE"] = "1"
```

Comment:
we can check the torch version, something like "if is_torch_equal_or_newer("2.8.0")"